ads/google/a4a/test/test-utils.js

```diff
@@ -148,15 +148,15 @@ describe('Google A4A utils', () => {
         setupForAdTesting(fixture);
         let url;
         const headers = {
-          get: function(name) {
+          get(name) {
```

Comment:
@cramforce do we prefer this? I think the eslint autofix redid all of these, i'll revert if these have perf implications